gbpservice/nfp/orchestrator/modules/device_orchestrator.py

```diff
@@ -432,13 +432,30 @@ def _delete_network_function_device_db(self, device_id, device):
                                             advance_sharing_interfaces)
         self.nsf_db.delete_network_function_device(self.db_session, device_id)
 
+    def _get_network_function_info(self, device_id):
+        nfi_filters = {'network_function_device_id': [device_id]}
+        network_function_instances = (
+            self.nsf_db.get_network_function_instances(self.db_session,
+                                                       nfi_filters))
+        network_function_ids = [nf['network_function_id']
+                for nf in network_function_instances]
+        network_functions = (
+            self.nsf_db.get_network_functions(self.db_session,
+                {'id': network_function_ids}))
+        return network_functions, network_function_instances
```

Comment:
The part of the return value network_function_instances is not used
We can just return network_functions